modules/vactory_tender/src/Controller/TenderSid.php

```diff
@@ -97,15 +97,17 @@ public function getFileUrl($sid, Request $request) {
         }
         $data = $webform_submission->getData();
 
-        $node =  $this-> entityTypeManager->getStorage('node')->load($data['tender']);
+        $node = $this-> entityTypeManager->getStorage('node')->load($data['tender']);
```

Comment:
@meryemdibe 
Remove space after arrow `$this->entityTypeManager` instead of `$this-> entityTypeManager`